src/run.rs

```diff
@@ -98,8 +99,16 @@ fn run_impl() -> anyhow::Result<i32> {
     // difference matters too much.
     let mut parallelism = usize::from(std::thread::available_parallelism()?);
 
+    let mut build_filename = "build.ninja".to_string();
```

Comment:
Maybe create this down by where it's used?

Reply:
done